2-multitenant/modules/env_baseline/outputs.tf

```diff
@@ -64,3 +64,10 @@ output "cluster_type" {
   description = "Cluster type"
   value       = var.cluster_type
 }
+
+output "cluster_service_accounts" {
+  description = "The service accounts to default running nodes as if not overridden in node_pools."
+  value = [
+    for value in merge(module.gke-standard, module.gke-autopilot) : value.service_account
```

Comment:
I don't recall if `merge()` also does `distinct()` for lists?  If not, probably want to `distinct()` the result.

Reply:
merge only works for maps/objects, and in the case of duplicate keys: "If more than one given map or object defines the same key or attribute, then the one that is later in the argument sequence takes precedence." 

in our case I think "module.gke-standard" would be mutually exclusive with "module.gke-autopilot"

Reply:
Agreed, the later precludes it mattering.